app/components/status_card/style.scss

```diff
@@ -8,13 +8,13 @@
   @include govuk-media-query($from: desktop) {
     padding: govuk-spacing(4);
   }
- 
+
   margin-bottom: govuk-spacing(3);
```

Comment:
```suggestion
```

I think we need to remove this as we're doubling up on spacing with `gap: govuk-spacing(4)`

Reply:
thanks, pushed up a fix